### PR TITLE
feat: added Greenhouse types

### PR DESCRIPTION
**Pull request by erikmejias**
Could be used to standardize some of our API usage.

### CODE REVIEW

File: src/types/index.ts [hunk 1]
```diff
@@ -6,3 +6,16 @@ export type { PartialExcept } from './PartialExcept';
 export type { PickPartial } from './PickPartial';
 export type { PickRequired } from './PickRequired';
 export type { RequiredExcept } from './RequiredExcept';
+export {
```
Comment (alexiglesias93) [inline src/types/index.ts:9]: I belive this can be just:
```typescript
export * from './apis/Greenhouse';
```

**Comment by alexiglesias93** [comment]
I belive you can do it by doing something like:
```typescript
export * as GreenHouse from './apis/Greenhouse';
```

**Comment by erikmejias** [comment]
Even better, was almost wrapping it using a `namespace` construct.

**Comment by alexiglesias93** [comment]
namespaces are considered deprecated in favor of using ESM import/export

**Comment by alexiglesias93** [comment]
Looks good!

**Comment by erikmejias** [comment]
Could you merge + publish?

**Pull request merged**